src/widget/OktaSignIn.js

```diff
@@ -163,6 +163,9 @@ var OktaSignIn = (function () {
      * @returns {Promise} - Returns a promise for an object containing the transaction information
      */
     function getTransaction(stateToken) {
+      if (!stateToken) {
+        throw 'A state token is required.';
```

Comment:
In the past, we've neglected language support for thrown error messages. Should we add a todo item for making sure these are localized?